lib/results.js

```diff
@@ -144,7 +144,7 @@ function encodeResult(res, count) {
     output += (res.ok ? 'ok ' : 'not ok ') + count;
     output += res.name ? ' ' + res.name.toString().replace(/\s+/g, ' ') : '';
 
-    if (res.skip) output += ' # SKIP';
+    if (res.skip) output += ' # SKIP' + ((typeof res.skip === 'string') ? ' ' + res.skip : '');
```

Comment:
what happens if the message contains a newline?

Let's add some test coverage for that.